packages/contentstack-import-setup/messages/index.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@shafeeqd959 what purpose we used this empty object file.

Reply:
Will add messages in future commits